Seneca-CDOT/telescope
# Remove firebase.json file from config

## Issue This PR Addresses
Fix #3089 


## Type of Change




## Description
 Removed firebase.json file from config


## Steps to test the PR



- Go to my repo
- Select 3089 branch
- Go to config 
- See that firebase.json was removed

## Checklist
